### PR TITLE
DEVO-860 New Jenkins

### PR DESCRIPTION
**Pull request by vitalykorolev**
### Description

This update to Jenkinsfile and makefile will allow us to use the new Jenkins instance.

#### Checklist: 

-  ##### Owner:

- [x] JIRA_ID as part of branch/PR name
- [x] Rebase the branch with upstream
- [x] Squashed all commits into a single commit
- [x] Added Tests
  
- ##### Reviewer:

- [ ] Reviewed Tests
- [ ] Added to Release Wiki/Jira

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -38,8 +39,8 @@ test: structure-test docker-tests
 # push docker images to mlregistry.marklogic.com
 #***************************************************************************
 push-mlregistry:
-	docker tag ${REPONAME}/marklogic-server-centos:${version} ml-docker-dev.marklogic.com/${repoDir}/marklogic-server-centos:${version}
-	docker push ml-docker-dev.marklogic.com/${repoDir}/marklogic-server-centos:${version}
+	docker tag ${REPONAME}/marklogic-server-centos:${version} ${docker_regestry}/${repoDir}/marklogic-server-centos:${version}
+	docker push ${docker_regestry}/${repoDir}/marklogic-server-centos:${version}
```
Comment (rwinieski) [inline Makefile:43]: Typo here as well:
regestry = registry ?

File: Makefile [hunk 2]
```diff
@@ -1,9 +1,10 @@
-version?=10-internal
+version?=internal
 package?=MarkLogic.rpm
-REPONAME=marklogic-centos
+REPONAME=local-dev
+docker_regestry?=ml-docker-db-dev-tierpoint.bed-artifactory.bedford.progress.com
 repoDir=marklogic
 docker_build_options=--compress
-test_image?=ml-docker-dev.marklogic.com/${repoDir}/marklogic-server-centos:${version}
+test_image?=${docker_regestry}/${repoDir}/marklogic-server-centos:${version}
```
Comment (rwinieski) [inline Makefile:7]: I guess there is a typo here:
regestry = registry ?

**Review by rwinieski — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (vitalykorolev) [reply Makefile:43]: Yes, and in three other places.

File: Makefile (hunk 2 above)
Comment (vitalykorolev) [reply Makefile]: Fixed.

**Review by rwinieski — APPROVED** [review]
Not forget to squash before merging

**Comment by vitalykorolev** [comment]
The Jenkins job on old will fail because it doesn't have access to the new data center.

**Pull request merged**